### ISSUE TITLE
Combine with ASL

### ISSUE THREAD

**Issue by catherinezll95** [Kevinz-code/CSRA#1]
Thanks for you good work. When I combine MHA with ASL(https://github.com/Alibaba-MIIL/ASL), I found the result will decrease. Do you try to combine your work with ASL？

**Comment by Kevinz-code** [comment]
Hi,
Thanks for reading.

Actually, we have tried to combine MHA with various of loss functions (e.g. Focal, ASL, etc). But they rarely bring about performance gain (mostly decrease).  Still, ASL only did improve the result.

It can be hard to increase the performance by combining different methods when the mAP is already very high. For example, 94.7mAP (in our paper) might be the upper bound of VOC2007 with 448 resolution and ResNet-101 pytorch official backbone. The most possible way to improve it can be larger resolution, data aug, etc.

**Comment by catherinezll95** [comment]
Thanks~😊
